src/core/exec-invoke.c

```diff
@@ -4212,10 +4212,52 @@ static bool exec_context_need_unprivileged_private_users(
                !strv_isempty(context->no_exec_paths);
 }
```

Comment:
i am pretty sure exec_context_need_unprivileged_private_users() should return true if namespaces are delegated in --user mode, no?